.github/workflows/lock-versions.yml

```diff
@@ -6,7 +6,7 @@ on:
           description: 'List of branches to lock versions (ordered, comma separated)'
           required: true
           # 7.17 was intentionally skipped because it was added late and was bug fix only
-          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10,8.11,8.12,8.13,8.14'
+          default: '8.9,8.10,8.11,8.12,8.13,8.14'
```

Comment:
We still have 6 versions here.

Reply:
Current - 5, which is inclusive of current version making it 6, was the understanding. 
We will be releasing 8.11,8.12,8.13,8.14 and maintaining 2 more versions 8.9,8.10.

Reply:
Happy to trim one more ;) although!

Reply:
Quoting a slack confirmation! 

`I’d prefer 5 but I’m on with 6 total` --> Resolving this